cypress/integration/home_page_spec.js

```diff
@@ -1,15 +1,38 @@
+// to start the test run npx cypress open into your terminal
 describe('Home page user flow', () => {
     beforeEach(() => {
         cy.onLoad()
     });
-// this first test is skipet because we don't have all the JSX elements yet
-    it.skip('Should contain the Naviagtion bar with all elements ', () => {
-        cy.get()
+// 
+    it('Should contain a Header (the Navigation bar) with a How to and a Image ', () => {
+        cy.get('.header')
           .should('be.visible')
-          .get()
-          .contains()
+          .get('.header-image')
+          .should('be.visible')
+          .get('.how-to')
+          .contains('How To')
```

Comment:
Great job adding the new elements

Reply:
Thanks for capturing all of the things here.